api/base/serializers.py

```diff
@@ -58,6 +58,40 @@ def is_anonymized(request):
     return website_utils.check_private_key_for_anonymized_link(private_key)
 
 
+class ShowIfVersion(ser.Field):
+    """
+    Skips the field if the specified request version is not within the field's allowed versions.
```

Comment:
I can certainly make the comment here more explicit. And actually, I'm not sure why I made `min_version` inclusive and `max_version` exclusive... they don't need to be at all. If they are both exclusive, do you feel that `min_version` and `max_version` are still appropriate names? Or would parameters like `earliest_supported_version`/`last(latest?)_supported_version` be better?